pawn/Resources/Callbacks/OnPlayer/OnPlayerText.pwn

```diff
@@ -8,6 +8,11 @@
  * @param playerid Id of the player who typed the text.
  * @param text The text the player typed.
  */
+ new QuitJokes[] = {
```

Comment:
Would you mind moving the variable to above the comment block? The comment block applies to OnPlayerText(), so it would be nice if it's immediately in front of it.